### ISSUE TITLE
wmic os get caption / wmi error.

### ISSUE THREAD

**Issue by smtszk** [wazuh/wazuh#312]
Hi,

I have faced a problem when I tried to install wazuh agent to Windows 2012 and Windows 2016 server. 

Normally, wazuh agent gets OS name during installing and moves on but I realized that there is an error when...

`wmic get os caption` command called.

My server returns an error:

<img width="1103" alt="screen shot 2017-12-13 at 12 47 43" src="https://user-images.githubusercontent.com/10809726/33932436-fb18d616-e003-11e7-90cf-d156a1712739.png">

and even if wazuh agent is installed, it's not working when I start the agent.

But when I try this:

`wmic os get caption /value`

<img width="1098" alt="screen shot 2017-12-13 at 12 48 22" src="https://user-images.githubusercontent.com/10809726/33932447-0433b518-e004-11e7-8afa-aa2fba97b293.png">

It works.

For some reason wmic can’t format output as a table (the same error you can get with /format: table switch and get rid of it with /format:list) (http://trukhanov.com/2014/02/wmic-exception-occured/)

Anything related to this?

**Comment by PSanchezR** [comment]
Hi smtszk,

I've tried to reproduce your issue on windows 2016 server Standar version and i've found it works for me:

![wmic output](https://user-images.githubusercontent.com/5520813/33946923-5a5227ee-e023-11e7-87ea-4c722e1866dc.png)

Over windows 2012 server R2 Standar Version i've got:

![w12](https://user-images.githubusercontent.com/5520813/33947008-999fb9b6-e023-11e7-96b2-f0c3326b0a05.png)

Could you try to run "wmic os get codeset" in order to get your OS codification?. Sometimes we've had similar issues related with the OS characters codification.

The code that you will get should be matched in the following link: 
https://en.wikipedia.org/wiki/Windows_code_page#List


Best regards.

**Comment by smtszk** [comment]
Thank you so much,

I tried but still doesn't work without "/value", It occurs Windows 2012 and Windows 2016 but not Windows 2008.

I installed another local windows wm and it worked.

<img width="590" alt="screen shot 2017-12-13 at 20 17 47" src="https://user-images.githubusercontent.com/10809726/33952429-cff0d1f2-e042-11e7-8f5c-e2111cf38059.png">

But this is interesting, I looked at WMI logs and found nothing.
